subsystem-drive/test/steer-modes_test.cpp

```diff
@@ -36,15 +36,15 @@ namespace sjsu::drive
             arguments = SteerModes::DriveSteering(commands);
             CHECK_EQ(arguments.back.hub.angle, 0);
             CHECK_EQ(arguments.back.hub.speed, 0);
-            CHECK_EQ(arguments.back.steer.angle, 0);
+            CHECK_EQ(arguments.back.steer.angle, 116);
```

Comment:
these should be static constexpr variables

Reply:
How would I re-implement them as such?

Reply:
sjsu::drive::RulesEngine::kMaxSpeed, would require including the rules engine header file